### ISSUE TITLE
Can't end meeting

### ISSUE THREAD

**Issue by gcrickman** [ParabolInc/parabol#4912]
## Issue - Bug

A [user](https://app.hubspot.com/contacts/3888472/contact/26069551) 🔒  writes: 

> First off: thanks for providing this awesome tool. It really boosts the productivity of my team & is one of the few tools where even our frontend developers are in love with the look and feel.
> However, I have a problem which is why I am contacting you here: We had a meeting in our team (sprint poker #2) and the facilitator forgot to end the meeting. I noticed some days later and wanted to end the meeting, but unfortunately none of us can enter the meeting anymore (to click "end meeting"): If we click on the meeting, the browser wants to go to the meeting but is immediately redirected to the teams page (https://action.parabol.co/meetings). This happens for different people and with different browsers, etc.
> How can we end this meeting?
>

**Comment by tiffanyhan** [comment]
server errors: 

```
2021-04-26T18:06:54.382309078Z app[web.1]: SEND TO SENTRY [Error: Argument "teamId" of non-null type "ID!" must not be null.] undefined
2021-04-26T18:06:54.521302715Z app[web.1]: SEND TO SENTRY [Error: Cannot destructure property 'dimensions' of 'o' as it is undefined.] undefined
```

https://sentry.io/organizations/parabol/issues/2364404325/?project=107915&query=is%3Aunresolved
https://sentry.io/organizations/parabol/issues/2221327157/?project=107915&query=is%3Aunresolved

dimensions destructure error seems to be thrown here: 

https://github.com/ParabolInc/parabol/blob/774c4cf2dfbceaa2d56910583e325080c55deae0/packages/server/graphql/types/EstimateStage.ts#L64

i checked in the db and no template with that id exists

**Comment by tiffanyhan** [comment]
i don't see any sentry errors during the duration of the meeting that indicate a failure of any db write

https://sentry.io/organizations/parabol/issues/?end=2021-04-12T14%3A14%3A59&project=107196&start=2021-04-12T14%3A08%3A00&utc=true

**Comment by tiffanyhan** [comment]
`catchAndLog` util should be used for all pg queries

**Comment by tiffanyhan** [comment]
there are 242 distinct templateRefIds in rethink, and only 40 of them map to actual objects in pg

**Comment by tiffanyhan** [comment]
meetings with bad templateRefId references started from Thu Feb 25 2021 23:48:15 GMT+00:00 to present

**Comment by tianrunhe** [comment]
What's happened was that during the meeting, the facilitator disconnected a while, triggering an edge case where his stage was out of sync with rest of the team. I manually updated the `faciliatorStageId` of their meeting in the DB and can confirm they will be able to end the meeting if they wish.

Also replied back to the user in the thread and closed the ticket.

**Comment by tiffanyhan** [comment]
thanks for finding and fixing the root cause! i'm still stuck on the missing templateRefId thing, esp. since lots of meetings have bad references but i'm not sure the impact of it since it wasn't related to them not being able to end the meeting.

**Comment by tianrunhe** [comment]
> thanks for finding and fixing the root cause! i'm still stuck on the missing templateRefId thing, esp. since lots of meetings have bad references but i'm not sure the impact of it since it wasn't related to them not being able to end the meeting.

Hey @tiffanyhan, sorry I forgot to mention in my reply regarding the templateRef thing. I did take a look at it when I started to investigate but I couldn't find the differences you have seen. I see the same number of TemplateRef between Rethink and PG:

<img width="943" alt="Screen Shot 2021-05-04 at 3 28 11 PM" src="https://user-images.githubusercontent.com/1879975/116973055-8d6f2f00-acee-11eb-9dbd-57e60349f162.png">


<img width="491" alt="Screen Shot 2021-05-04 at 3 36 19 PM" src="https://user-images.githubusercontent.com/1879975/116973024-7cbeb900-acee-11eb-8675-7dab45407ca9.png">

Did I miss anything here?
